website/content/v1.8/kubernetes-guides/network/deploying-cilium.md

```diff
@@ -357,8 +357,11 @@ We can leverage this to our advantage to install Cilium by using an inline manif
                 - name: KUBERNETES_SERVICE_PORT
                   value: "6443"
                 command:
-                  - cilium
+                  - helm
                   - install
+                  - cilium
```

Comment:
is it a typo or should it be two times?

I would want some input from @leppeK on this change, as he brought this change initially

Reply:
I believe it is correct.
helm install [NAME] [CHART] [flags]

Reply:
For helm it is correct, unfortunately helm leaves traces of the release which you might not want.

I'll see if I can convince Cilium to provide an official container image for all platforms

Reply:
I think the comments already indicate that you can use this method to do anything, if you want to use this to install something else using helm or whatever it works as well.


A use case could be installing your CNI and/or ArgoCD at bootstrap time

Reply:
Isn't it the same as manually installing using helm after bootstrap?

Reply:
Doing stuff at bootstrap means you can do it completely declaratively and in one go and at the right time... Great for reproducibilty